librz/analysis/arch_profile.c

```diff
@@ -201,6 +205,7 @@ RZ_API bool rz_arch_profiles_init(RzArchTarget *t, const char *cpu, const char *
 	if (!is_cpu_valid(cpu_dir, cpu)) {
 		if (arch) {
```

Comment:
move also this check at the beginning, and if it fails, then return false.

Reply:
I've added this along with the check of `path`. Please see if that's okay.